deploy/k3s/inference_deployment/inference_image_warmer.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: image-cache-warmer
+spec:
+  template:
+    spec:
+      imagePullSecrets:
```

Comment:
Can you also add the aws s3 sync step that should be present in the inference deployment template now?